File: All_Psets_Solve_Files/mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
//storing hieght in variable
    int h;
//prompt user for hiegt
    do
    {
        h = get_int("Hieght: ");
    }
    while (h < 1 || h > 8);
//loop for lines
    for (int i = 0; i < h; i++)
  {
      //loop for spaces
      for (int d = h-1; d > i; d--)
      {
            printf(" ");
      }
      //loop for hashes
      for (int j = -1; j < i; j++)
      {
         printf("#");
      }
         printf("  ");
    for (int k = -1; k < i; k++)
    {
        printf("#");
    }
        printf("\n");
  }
}

File: All_Psets_Solve_Files/sentimental-mario-less/mario.py
# function to get hieght value
from cs50 import get_int

# variable to store hieght
h = get_int('Height: ')
# loop to check that the value is correct
while h < 1 or h > 8:
    # ask again
    h = get_int('Height: ')

# loop to print hashes
for i in range(1, h+1):
    # print space
    print(' ' * (h - i), end='')
    print('#' * i)
